powered-by-art-blocks-pbab-onboarding/pbab-101/README.md

```diff
@@ -15,6 +15,7 @@ A high level process-map for PBAB onboarding.
         * To create a new project shell, you should use the `addProject` method of your newly deployed PBAB Core Contract. This can be done by connecting to the contract via Etherscan.
 4.  Your team will integrate a custom web frontend with your deployed PBAB smart contracts (e.g. **implementing their own purchase + display flow**) and with the Art Blocks API as needed. An example of frontend purchase flow logic is provided here as a reference for integrating partners:
 
+<details><summary>example</summary>
```

Comment:
I think we should probably make the `summary` tag here something more descriptive than the default of "example".

Reply:
Oh wait, this is literally a code example 😂 ignore my comment.